Documentation/quickstart/install.rst

```diff
@@ -219,7 +219,7 @@ Apache NuttX is actively developed on GitHub. There are two main repositories, `
   .. tab:: Download stable release
   
     Go to `releases <https://nuttx.apache.org/download/>`_ and choose a version to download. The following
-    example uses version 10.1.0:
+    example uses version 12.0.0:
```

Comment:
Why don't you use the last 12.2.1 release?

Reply:
Hi Alan, done.